### DISCUSSION TITLE
Doctrine BUG

### DISCUSSION THREAD

**Discussion by Bilel69500** [symfony/symfony#46200] [Q&A] [answered]
### Symfony version(s) affected

5.4.2

### Description

Hello, 
I have this error message 
![Capture d’écran 2022-04-28 à 00 25 56](https://user-images.githubusercontent.com/55830730/165641505-e82682b6-26c9-4421-a2b4-31e609013507.png)

### How to reproduce

after install doctrine on symfony i have this message error

**Comment by fd6130** [comment, marked as answer]
How about update your symfony version to latest 5.4.8 and also reinstall doctrine-bundle `composer require doctrine/doctrine-bundle`?

**Comment by Bilel69500** [comment]
Hello,
I upgrade symfony to symfony 6.0 is okay 
thx
